### ISSUE TITLE
tooltip flickers when hovering over some icons on gear page

### ISSUE THREAD

**Issue by Harmony222** [Harmony222/CS361_Kitty_Kontingent#1]
Issue: some of the tooltips flicker when hovering over the icon next to a gear item

Steps to reproduce: I had a difficult time reproducing this bug. I initially thought perhaps it was due to the length of some of the tooltips since there was one particular gear item (trekking poles) that was semi-consistent in producing the bug and when I modified some of the trekking pole data it happened to fix the issue. Later I was working on the icons again and it started happening more frequently, but again without a way to directly reproduce the bug.

Possible solution: it appears this is a known issue with Bootstrap and I found some helpful information from stack overflow:
https://stackoverflow.com/questions/14326724/twitter-bootstrap-tooltip-flickers-when-placed-on-top-of-button-but-works-fine%20*/

From the link "tooltip position attempts to automatically change when a parent container has overflow" and there are several solutions provided. I found the most straightforward to be adding this to CSS:

.tooltip {
  pointer-events: none;
}

Bottom line: if you are having problems with tooltips flickering, try adding the above code to your CSS
